src/utils/HypervBoxUtil.js

```diff
@@ -31,19 +28,19 @@ var HypervBox = {
   */
 
   hasAdminRights: function() {
-    return util.execFile([this.command(), 'Get-VMHostSupportedVersion']).then(stdout => {
+    return util.execFile([this.command(), 'Get-VMHost']).then(stdout => {
```

Comment:
If I run the command from a non elevated powershell I get this output: 

```
get-vmhost : Sie besitzen nicht die erforderliche Berechtigung für diese Aufgabe. Wenden Sie sich an den Administrator
der Autorisierungsrichtlinie für Computer "PC-MARIO".
In Zeile:1 Zeichen:1
+ get-vmhost
+ ~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Get-VMHost], VirtualizationException
    + FullyQualifiedErrorId : Unspecified,Microsoft.HyperV.PowerShell.Commands.GetVMHost
```

So for me there is no text "invalidoperation" ...